### PR TITLE
Bugs/8109

### PR DESCRIPTION
**Pull request by bradhead**
## Status
**Ready**

## Fixes or Implements [AB#8109](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8109)????
- [ ] Enhancement
- [x] Bug
- [ ] Documentation

## Description:
A sentences or two describing the overall goals of the pull request's commits.

## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

### Steps to Reproduce:
Smelly code - stylecop hates me! I fixed up common auth code to remove smells and cleaned up throwing exceptions and Contract checking on nulls.  

### UI Changes
NO

### Browsers Tested
- [x] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments
Re-ran user delete test manually on Local, which included running JobScheduler, the front end and KeyCloak local.

### CODE REVIEW

File: Apps/Common/src/Models/Patient.cs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿//-------------------------------------------------------------------------
-// Copyright © 2020 Province of British Columbia
+// Copyright © 2019 Province of British Columbia
```
Comment (FuriousLlama) [inline Apps/Common/src/Models/Patient.cs]: ?

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_407)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_407)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_407)

**Pull request merged**